src/helper.php

```diff
@@ -33,6 +33,8 @@ function dd()
             echo (PHP_SAPI == 'cli' ? strip_tags($string) : $string);
         }, func_get_args());
 
+        echo "\n";
```

Comment:
could you please use this before? `PHP_SAPI == 'cli' ? strip_tags($string) . PHP_EOL : $string`